[CI] FullClusterRestartIT testRollupAfterRestart {cluster=OLD} failing

**Build Scans:**
- [elasticsearch-periodic #4523 / openjdk17_7.17.25_java-fips-matrix-bwc](https://gradle-enterprise.elastic.co/s/rn2jy3ngizrdi)
- [elasticsearch-periodic #4463 / 7.15.2_bwc](https://gradle-enterprise.elastic.co/s/yniqxhkn5jhvm)

**Reproduction Line:**
```
./gradlew ":x-pack:qa:full-cluster-restart:v7.17.25#bwcTest" -Dtests.class="org.elasticsearch.xpack.restart.FullClusterRestartIT" -Dtests.method="testRollupAfterRestart {cluster=UPGRADED}" -Dtests.seed=678A207D9C9D6903 -Dtests.bwc=true -Dtests.locale=da -Dtests.timezone=Pacific/Honolulu -Druntime.java=17 -Dtests.fips.enabled=true
```

**Applicable branches:**
8.16

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('testRollupAfterRestart%20%7Bcluster%3DOLD%7D'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.xpack.restart.FullClusterRestartIT'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure Message:**
```
java.lang.AssertionError: null
```

**Issue Reasons:**
- [8.16] 2 failures in test testRollupAfterRestart {cluster=OLD} (0.2% fail rate in 853 executions)
- [8.16] 2 failures in pipeline elasticsearch-periodic (15.4% fail rate in 13 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/x-pack/qa/full-cluster-restart/src/javaRestTest/java/org/elasticsearch/xpack/restart/FullClusterRestartIT.java
+++ b/x-pack/qa/full-cluster-restart/src/javaRestTest/java/org/elasticsearch/xpack/restart/FullClusterRestartIT.java
@@ -435,6 +435,7 @@ public class FullClusterRestartIT extends AbstractXpackFullClusterRestartTestCas
 
             final Request bulkRequest = new Request("POST", "/_bulk");
             bulkRequest.setJsonEntity(bulk.toString());
+            bulkRequest.setOptions(RequestOptions.DEFAULT.toBuilder().setWarningsHandler(fieldNamesFieldOk()));
             client().performRequest(bulkRequest);
 
             // create the rollup job

